exhibitor-core/src/main/java/com/netflix/exhibitor/core/Exhibitor.java

```diff
@@ -135,7 +136,14 @@ public Exhibitor(ConfigProvider configProvider, Collection<? extends UITab> addi
         processMonitor = new ProcessMonitor(this);
         autoInstanceManagement = new RepeatingActivityImpl(log, activityQueue, QueueGroups.MAIN, new AutomaticInstanceManagement(this), getAutoInstanceManagementPeriod());
 
-        remoteInstanceRequestClient = new RemoteInstanceRequestClientImpl(arguments.remoteConnectionConfiguration);
+        if(arguments.httpsConfiguration.getServerKeystorePath() != null)
```

Comment:
When I discussed the overall design with JP, we agreed that setting `serverKeystorePath` enables TLS for both server and client. This is the current project document: https://docs.google.com/document/d/1JKkDWPzu_lPiF7hAMtVNptEprunClTBwzw2S-9WOVZI